### PR TITLE
Add support for Conrad S3318P

### PR DESCRIPTION
**Pull request by mnhauke**

### CODE REVIEW

**Comment by enavarro222** [comment]
Hi !

Is it for that sensors ?
http://www.conrad.com/ce/en/product/672868/S014-Additional-Wireless-Sensor-For-S3318P-Weather-Station

If it's the case, they are very similar to some cheap ones I get from Aliexpress, see https://github.com/merbanan/rtl_433_tests/blob/master/tests/prologue/02/front.jpg

However the protocol seems to be different.

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -64,6 +64,7 @@ add_executable(rtl_433
 	devices/waveman.c
 	devices/wt450.c
 	devices/x10_rf.c
+    devices/s3318p.c
```
Comment (merbanan) [inline src/CMakeLists.txt]: spaces -> tab

File: src/devices/s3318p.c [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    humidity = (uint8_t)(((bb[1][3] & 0x0F) << 4) | ((bb[1][3] & 0xF0) >> 4));
+    button = (uint8_t)(bb[1][4] >> 7);
+    battery_low = (uint8_t)((bb[1][4] & 0x40) >> 6);
+    channel = (uint8_t)(((bb[1][1] & 0x30) >> 4) + 1);
+    sensor_id = (uint8_t)(bb[1][0]);
+
+    temperature_with_offset = (uint16_t)(((bb[1][2] & 0x0F) << 8) | (bb[1][2] & 0xF0) | (bb[1][1] & 0x0F));
+    temperature_f = (float)((temperature_with_offset - 900) / 10.0);
+    temperature_c = fahrenheit2celsius(temperature_f);
+
```
Comment (merbanan) [inline src/devices/s3318p.c]: You are supposed to report the data from the sensor as you got it. So remove this line and change the output to temperature_f instead. There is a command line option that will convert the data properly for you.

**Comment by merbanan** [comment]
Looks good, only minor changes and I'll merge the code.

**Comment by merbanan** [comment]
Oh, I forgot. I need signal samples also. Submit them to the rtl_433_tests repository.

**Comment by mnhauke** [comment]
@enavarro222
It's exactly that sensor. The housing looks very similar to the one from your sensors.

**Comment by mnhauke** [comment]
@merbanan 
I'll add some signal captures tomorrow to the rtl_433_tests repository.

**Comment by mnhauke** [comment]
I've added signal captures, protocol information and pictures to the rtl_433_tests repository
https://github.com/merbanan/rtl_433_tests/commit/e9d7d12943cbebf3170ac204605c50bfe2152ca5

**Comment by merbanan** [comment]
Thanks

**Pull request merged**